cost-analyzer/values.yaml

```
@@ -2578,6 +2578,10 @@ kubecostAggregator:
   #
   # default: 91
   etlDailyStoreDurationDays: 91
+  # disabling the hourly stores can improve ingestion performance, but is
+  # not currently supported due to the frontend defaulting to hourly queries
+  # on certain pages.
```

```suggestion
  # How much hourly data to ingest from the federated store bucket, and how much
  # to keep in the DB before rolling the data off.
  #
  # In high scale environments setting this to `0` can improve performance if hourly
  # resolution is not a requirement.
  #
  # default: 49
```

Minor comment suggestion